clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/converters/ResizeGoogleServerGroupAtomicOperationConverter.groovy

```diff
@@ -38,9 +41,14 @@ import org.springframework.stereotype.Component
 @Component("resizeGoogleServerGroupDescription")
 class ResizeGoogleServerGroupAtomicOperationConverter extends AbstractAtomicOperationsCredentialsConverter<GoogleNamedAccountCredentials> {
 
-  @Autowired
```

Comment:
Seems like we still need `@Autowired` here, and for the other new member variables.

Reply:
I am not sure whether here it is required.

Reply:
Done